tests/karate/src/test/java/be/utils/simpleScenarios.feature

```diff
@@ -0,0 +1,65 @@
+@ignore @report=false
+  Feature: create lao
+    @name=valid_lao
+    Scenario: Creates valid lao
+
+      * string laoCreateReq  = read('classpath:data/lao/valid_lao_create.json')
+      * eval frontend.send(laoCreateReq)
+      * frontend_buffer.takeTimeout(timeout)
+
+      * def subscribe =
```

Comment:
Is there a reason why this string is defined here rather than read as usual from a json file ?

Reply:
subscribe and catchup are used only here, and since this is a scripting file, I think it is okay to leave it as it is since these too messages will be used only in conjunction with the underlying valid lao.